pkg/declarativeconfig/transform/role.go

```diff
@@ -38,11 +40,13 @@ func (r *roleTransform) Transform(configuration declarativeconfig.Configuration)
 	}
 
 	roleProto := &storage.Role{
-		Name:            roleConfig.Name,
-		Description:     roleConfig.Description,
-		PermissionSetId: declarativeconfig.NewDeclarativePermissionSetUUID(roleConfig.PermissionSet).String(),
-		AccessScopeId:   declarativeconfig.NewDeclarativeAccessScopeUUID(roleConfig.AccessScope).String(),
-		Traits:          &storage.Traits{Origin: storage.Traits_DECLARATIVE},
+		Name:        roleConfig.Name,
+		Description: roleConfig.Description,
+		PermissionSetId: stringutils.FirstNonEmpty(accesscontrol.DefaultPermissionSetIDs[roleConfig.PermissionSet],
```

Comment:
I don't see a reason why for permission sets and access scopes we would use different implementation(map vs function with `switch` inside). Perhaps we could put Access scopes in the map as well?